extensions/lifecycle/tasks/LifecycleColdStatusArchiveTask.js

```diff
@@ -40,6 +42,38 @@ class LifecycleColdStatusArchiveTask extends LifecycleUpdateTransitionTask {
         });
     }
 
+    /**
+     * Requests the deletion of a cold object by pushing
+     * a message into the cold GC topic
+     * @param {string} coldLocation cold location name
+     * @param {ColdStorageStatusQueueEntry} entry entry received
+     * from the cold location status topic
+     * @param {Logger} log logger instance
+     * @param {function} cb callback
+     * @return {undefined}
+     */
+    _deleteColdObject(coldLocation, entry, log, cb) {
+        const coldGcTopic = `${this.lcConfig.coldStorageGCTopicPrefix}${coldLocation}`;
+        const gcMessage = JSON.stringify({
+            bucketName: entry.target.bucketName,
+            objectKey: entry.target.objectKey,
+            objectVersion: entry.target.objectVersion,
+            archiveInfo: entry.archiveInfo,
+            requestId: entry.requestId,
+        });
+        this.coldProducer.sendToTopic(coldGcTopic, [{ message: gcMessage }], err => {
+            if (err) {
+                log.error('error sending cold object deletion entry', {
+                    error: err,
+                    entry: entry.getLogInfo(),
+                    method: 'LifecycleColdStatusArchiveTask._deleteColdObject',
+                });
+                return cb(err);
+            }
+            return cb(new SkipMdUpdateError('cold object deleted'));
```

Comment:
do we need to return an error here?
the "error" is actually handled, and it requires you to ignore that error at line 161 : so it may be best to just return `null` here instead...